src/contained-list/contained-list.stories.ts

```diff
@@ -0,0 +1,333 @@
+		</ibm-contained-list>
+	`
+});
+export const withActions = withActionsTemplate.bind({});
+
+const withIconsTemplate: Story<ContainedList> = () => ({
+	template: `
+		<ng-template #apple let-icon>
+			<svg ibmIcon="apple" size="16"></svg>
+		</ng-template>
+
+		<ng-template #wheat let-icon>
+			<svg ibmIcon="wheat" size="16"></svg>
+		</ng-template>
+
+		<ng-template #strawberry let-icon>
+			<svg ibmIcon="strawberry" size="16"></svg>
+		</ng-template>
+
+		<ng-template #fish let-icon>
+			<svg ibmIcon="fish" size="16"></svg>
+		</ng-template>
+
+		<ibm-contained-list label="List title">
+			<ibm-contained-list-item [icon]="apple">List item</ibm-contained-list-item>
+			<ibm-contained-list-item [icon]="wheat">List item</ibm-contained-list-item>
+			<ibm-contained-list-item [icon]="strawberry">List item</ibm-contained-list-item>
+			<ibm-contained-list-item [icon]="fish">List item</ibm-contained-list-item>
+		</ibm-contained-list>
+	`
+});
+export const withIcons = withIconsTemplate.bind({});
+
+const withInteractiveItemsTemplate: Story<ContainedList> = () => ({
+	template: `
+		<ibm-contained-list label="List title">
+			<ibm-contained-list-item [clickable]="true">
+				<ng-container ibmContainedListItemButton>List item</ng-container>
+			</ibm-contained-list-item>
+			<ibm-contained-list-item [clickable]="true" [disabled]="true">
+				<ng-container ibmContainedListItemButton>List item</ng-container>
+			</ibm-contained-list-item>
+			<ibm-contained-list-item [clickable]="true">
+				<ng-container ibmContainedListItemButton>List item</ng-container>
+			</ibm-contained-list-item>
+			<ibm-contained-list-item [clickable]="true">
+				<ng-container ibmContainedListItemButton>List item</ng-container>
+			</ibm-contained-list-item>
+		</ibm-contained-list>
+	`
+});
+export const withInteractiveItems = withInteractiveItemsTemplate.bind({});
+
+const withInteractiveItemsAndActionsTemplate: Story<ContainedList> = () => ({
+	template: `
+		<ng-template #action let-item>
+			<button
+				aria-label="Action"
+				ibmButton="ghost"
+				iconOnly="true">
+				<svg ibmIcon="subtract--alt" size="16" class="cds--btn__icon"></svg>
+			</button>
+		</ng-template>
+		<ibm-contained-list label="List title">
+			<ibm-contained-list-item [clickable]="true" [action]="action">
+				<ng-container ibmContainedListItemButton>List item</ng-container>
+			</ibm-contained-list-item>
+			<ibm-contained-list-item [clickable]="true" [action]="action">
+				<ng-container ibmContainedListItemButton>List item</ng-container>
+			</ibm-contained-list-item>
+			<ibm-contained-list-item [clickable]="true" [action]="action">
+				<ng-container ibmContainedListItemButton>List item</ng-container>
+			</ibm-contained-list-item>
+			<ibm-contained-list-item [clickable]="true" [action]="action">
+				<ng-container ibmContainedListItemButton>List item</ng-container>
+			</ibm-contained-list-item>
+		</ibm-contained-list>
+	`
+});
```

Comment:
Could you also add an expandable search component to this? React has it in their [storybook](https://react.carbondesignsystem.com/?path=/story/components-containedlist--with-interactive-items-and-actions), I THINK it should work, but I just want to double check.

Reply:
At the time I open the PR the new docs on search was incomplete, I will verify now and retry the implementation.

Reply:
I  think we are having some scss problems with search. Look at e86c763 to see how the component is placed on the story.

Reply:
It's expected, we can't do much about it. We would have to add `style: block` to the `cds-search` tag. Overall, it isn't a major issue & can be tackled by the users!